### DISCUSSION TITLE
Enhancing Material-React-Table for Improved Mobile View with Accordion Feature.

### DISCUSSION THREAD

**Discussion by ysrazsingh** [KevinVandy/material-react-table#883] [Ideas]
I have found the material-react-table quite beneficial and have incorporated it into all the projects I've worked on. However, I believe there is room for improvement, especially concerning its mobile view. Currently, when viewed on mobile devices, the table data requires horizontal scrolling to access all information. I suggest incorporating an accordion-style feature, similar to the example illustrated here: https://codepen.io/SitePoint/pen/xbvWQv

This table library is already impressive, and the implementation of such a feature would enhance its functionality even further.

**Comment by KevinVandy** [comment]
I don't think I will add a specific mobile feature, but can instead show how to customize MRT with a mobile friendly table body with a media query. It's already easily possible.

**Reply by dkelsen** [reply]
I would be curious about how you imagine the mobile friendly table to look like.
